test-exports/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Tests for the things exported by the langchain package",
   "main": "./index.js",
   "scripts": {
-    "test": "node ./index.js && node ./index.mjs && tsc --noEmit index.ts",
+    "test": "node ./index.js && node ./index.mjs && tsc",
```

Comment:
weird bug(?). if you run `tsc` with any argument it ignores the local `tsconfig.json`